### PR TITLE
Add readme with steps how to launch demo

### PR DESCRIPTION
**Pull request by andreyleskov**
I tried to launch demo on local machine and stumbled upon many issues. 
As a person with backend C# background it took me three hours to launch the demo and see how yrdt persist documents 
in mongo. 
This is a great project and it deserves a better onboarding and demo launch experience. 
Please find the updated README.md with steps it took me to have demo up and running plus minor code changes.

### CODE REVIEW

**Comment by SebastianStehle** [comment]
Sorry. I have forgotten to review your PR. I have only focused on the readme, because the other files also contain some test code. This needs to be cleaned up before something can be merged.

By default it should be possible to run the demo without MongoDB.

**Comment by andreyleskov** [comment]
@SebastianStehle done, please review. Only left ReadMe, for launching with in-memory persistence locally with dependencies install.

This project is C# and I can imagine backend-heavy folks like me can be attracted for realtime collaboration feature, but struggle to launch it due to lack of knowledge with frontend part.
A good option in my opinion is to include a compose file and dockerfile with all the build and setup steps, to launch in one command.

File: Demo/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+If you are launching this demo from scratch, follow the steps:
+
++ Install .Net 7.0
```
Comment (SebastianStehle) [inline Demo/README.md:3]: You can use .Net 8.0 here, we just have not updated yet.

**Review by SebastianStehle — CHANGES_REQUESTED** [review]

**Comment by SebastianStehle** [comment]
Closing because of inactivity

**Pull request closed without merging**